crates/sui-types/src/crypto.rs

```diff
@@ -1549,11 +1550,33 @@ impl<'a> VerificationObligation<'a> {
             pks,
             &self.messages.iter().map(|x| &x[..]).collect::<Vec<_>>()[..],
         )
-        .map_err(|e| SuiError::InvalidSignature {
-            error: format!(
+        .map_err(|e| {
+            let error_message = format!(
                 "Failed to batch verify aggregated auth sig {} pks: {:?} messages: {:?} sigs: {:?}",
                 e, &self.public_keys, &self.messages, &self.signatures
```

Comment:
while you are at it, can we change this to base64 encoded bls aggregated sig? also messages can be encoded too.